tests/unit/db/account_tests.py

```diff
@@ -450,6 +450,31 @@ def test_billing_data(self):
         finally:
             self.client.disconnect()
 
+    def test_set_year_without_month_for_billing_data(self):
+        """
+        Test raising an exception when retrieving billing data with only
+        year parameter
+        """
+        try:
+            self.client.connect()
+            year = 2016
+            self.assertRaises(CloudantException, self.client.bill, year)
+        finally:
+            self.client.disconnect()
+
+    def test_set_year_with_invalid_month_for_billing_data(self):
+        """
+        Test raising an exception when retrieving billing data with an
+        invalid month parameter
+        """
+        try:
+            self.client.connect()
+            year = 2016
+            month = 13
+            self.assertRaises(CloudantException, self.client.bill, year, month)
+        finally:
+            self.client.disconnect()
+
```

Comment:
I think it might also be useful to add a tests for:

1. When the month is supplied but the year is not
2. When year is invalid (like when a string is passed in)